src/vivarium_public_health/risks/base_risk.py

```diff
@@ -8,29 +8,10 @@
 
 """
 
-from typing import Any
+from vivarium_public_health.risks.exposre import Exposure
 
-import pandas as pd
-from vivarium import Component
-from vivarium.framework.engine import Builder
-from vivarium.framework.event import Event
-from vivarium.framework.population import SimulantData
-from vivarium.framework.randomness import RandomnessStream
-from vivarium.framework.resource import Resource
-from vivarium.framework.values import Pipeline
 
-from vivarium_public_health.risks.data_transformations import get_exposure_post_processor
-from vivarium_public_health.risks.distributions import (
-    ContinuousDistribution,
-    DichotomousDistribution,
-    EnsembleDistribution,
-    PolytomousDistribution,
-    RiskExposureDistribution,
-)
-from vivarium_public_health.utilities import EntityString, get_lookup_columns
-
-
-class Risk(Component):
+class Risk(Exposure):
```

Comment:
The implementation for risk is now in the Exposure class and the only thing we need here is the properties.